screens/ProfileScreen.tsx

```diff
@@ -22,32 +19,87 @@ export default function ProfileScreen({ navigation }: Props) {
   const colorMode = useAppSelector(selectColorMode);
   const dispatch = useAppDispatch();
 
-  const userId = 'user-2';
-  const user = mockedUsers.find((u) => u.id === userId);
+  const userId = 'user-1';
   const member = mockedMembers.find((m) => m.userId === userId);
-  // const members = useAppSelector(selectMembersByUser);
 
-  if (!user || !member) {
+  const [email, setEmail] = useState<string | null>(null);
+
+  useEffect(() => {
+    const user = auth.currentUser;
+    if (user) {
+      setEmail(user.email);
+    }
+  }, []);
+
+  useEffect(() => {
+    navigation.setOptions({
+      headerRight: () => (
+        <Button
+          onPress={handleLogout}
+          mode="text"
+          style={s.logoutButton}
+          labelStyle={{ fontSize: 20 }}
+        >
+          Logga ut
+        </Button>
+      ),
+    });
+  }, [navigation]);
+
+  if (!member) {
```

Comment:
tänk på hur du har kommit till denna sidan.

1. Först måste man logga in (user kommer alltid finnas)
2. Sedan ska man välja ett hushåll (selectedHousehold kommer alltid finnas + en member för dig på hushållet)
3. Sedan kan man komma till profilsidan (och då kommer alltid member finnas, så du kommer inte behöva kolla det basically.)